rasa/core/featurizers.py

```diff
@@ -41,15 +63,13 @@ def encode(self, state: Dict[Text, float]) -> np.ndarray:
         )
 
     @staticmethod
-    def action_as_one_hot(action: Text, domain: Domain) -> np.ndarray:
+    def action_as_index(action: Text, domain: Domain) -> Optional[int]:
```

Comment:
do we need this method? Can't we simply use `domain.index_for_action(action)` directly?